demosplan/DemosPlanStatementBundle/Repository/StatementRepository.php

```diff
@@ -1446,36 +1450,67 @@ public function getSubmittedOrAuthoredStatements(string $userId): array
         return $query->getResult();
     }
 
+    /**
+     * @param array $procedureIds the procedure IDs to get the number of original statements for
+     *
+     * @return array<string, int> unordered array with procedure IDs as keys and the count of corresponding
+     *                            original statements which are no placeholders and not deleted as values
+     */
+    public function getOriginalStatementsCounts(array $procedureIds): array
+    {
+        if ([] === $procedureIds) {
+            return [];
+        }
+
+        $queryBuilder = $this->createStatementsCountsQueryBuilder($procedureIds);
+        $queryBuilder->andWhere($queryBuilder->expr()->isNull('statement.original'));
```

Comment:
getOriginalStatementsCounts() and getStatementsCounts() seem to differ only in this one line, I think this should be handled differently to avoid code duplication.

Reply:
getOriginalStatementsCounts() was already existing.
I added getStatementsCounts(). To avoid code duplication I extracted the shared logic into createStatementsCountsQueryBuilder().

What do you mean with "handled differently"?
More reducing of the remaining code duplication?

Reply:
Yes I meant reducing the remaining code duplication.

Reply:
Pair reviewed - not a blocker for now. Could be refactored in a seperate PR.